Src/BootCamp.Chapter/TextBoxPrinter.cs

```diff
@@ -1,12 +1,137 @@
 ﻿using System;
-using System.Collections.Generic;
 using System.Text;
 
 namespace BootCamp.Chapter
 {
     public static class TextBoxPrinter
     {
-        // TODO: Implement.
-        public static string Print(object obj) => "";
+        public static string Print(object obj)
+        {
+            var atri = AttributesGetter.GetClassAttribute<TextableAttribute>(obj.GetType());
+            if (atri != null)
+            {
+                return MakePadding(obj.ToString(), atri.Padding, atri.Horizontal, atri.Vertical, atri.Corner);
+            }
+            else
+            {
+                return obj.ToString();
+            }
+        }
+
+        public static string MakePadding(string message, int padding = 0, char horizontal = '-', char vertical = '|', char corner = '+')
+        {
+            string[] splitmessage = message.Split(Environment.NewLine);
+            int height = splitmessage.Length + (padding * 2) + 1;
+            int width = 0;
+
+            foreach (string line in splitmessage) // checking which word is the longest to know how wide the padding should be.
```

Comment:
Instead of writing a comment, just split this to a private function ``FindLongestLineLength``